src/main/resources/assets/jei/lang/de_DE.lang

```diff
@@ -1,64 +1,88 @@
-config.jei.advanced.hideMissingModelsEnabled=Verstecke fehlende Item-Modelle
-config.jei.advanced.hideMissingModelsEnabled.comment=Items in der Item-Liste mit fehlenden Modellen werden automatisch verborgen.
-config.jei.advanced.debugModeEnabled=Debug-Modus
-config.jei.advanced.debugModeEnabled.comment=Fügt zufällige Strings hinzu, um bei der JEI-Entwicklung zu helfen. Nur für Entwickler bestimmt.
+config.jei.advanced.itemBlacklist=Aktiviere Blacklist
+config.jei.advanced.itemBlacklist.comment=Gegenstände, die nicht angezeigt werden sollen. Format: modId[:name[:meta]]. Der Listeneditiermodus entfernt und fügt automatisch neue Gegenstände zu dieser Liste hinzu.
+config.jei.advanced.hideMissingModelsEnabled=Verstecke fehlende Modelle
+config.jei.advanced.hideMissingModelsEnabled.comment=Gegenstände mit fehlenden Modellen werden automatisch in der Item-Liste verborgen.
+config.jei.advanced.colorSearchEnabled=Aktiviere Farbsuche
+config.jei.advanced.colorSearchEnabled.comment=Suche nach Gegenständen mittels ihrer Farbe und zeige die suchbaren Farben in Tooltips in der Item-Liste.
+config.jei.advanced.debugModeEnabled=Aktiviere Debug-Modus
+config.jei.advanced.debugModeEnabled.comment=Fügt tausende Test-Items und einige Debug-Rezepte hinzu. Nur für Entwickler bestimmt.
+config.jei.advanced.centerSearchBarEnabled=Zentriere Suchleiste
+config.jei.advanced.centerSearchBarEnabled.comment=Die JEI-Suchleiste wird unten in der Mitte des Bildschirms angezeigt.
 
 # Edit Mode
-gui.jei.editMode.description=JEI Item-Listeneditiermodus:
-gui.jei.editMode.description.hide=Strg-Klick zum Verbergen.
-gui.jei.editMode.description.show=Verborgen. Strg-Klick zum Zeigen.
-gui.jei.editMode.description.hide.wild=Strg-Rechtsklick, um mit einer Wildcard zu verstecken.
-gui.jei.editMode.description.show.wild=Verborgen durch Wildcard. Strg-Rechtsklick zum Zeigen.
+gui.jei.editMode.description=JEI-Listeneditiermodus:
+gui.jei.editMode.description.hide=Ausblenden mit STRG-Klick.
+gui.jei.editMode.description.show=Ausgeblendet. Einblenden mit STRG-Klick.
+gui.jei.editMode.description.hide.wild=Ausblenden per Wildcard mit STRG-Rechtsklick.
+gui.jei.editMode.description.show.wild=Ausgeblendet per Wildcard. Einblenden mit STRG-Rechtsklick.
+gui.jei.editMode.description.hide.mod.id=Ausblenden per ModId mit STRG-Shift-Klick.
+gui.jei.editMode.description.show.mod.id=Ausgeblendet per ModId. Einblenden mit STRG-Shift-Klick.
 
 # Recipe Categories
-gui.jei.category.craftingTable=Craftingrezepte
+gui.jei.category.craftingTable=Crafting-Rezepte
 gui.jei.category.smelting=Schmelzrezepte
-gui.jei.category.smelting.experience=%s XP.
+gui.jei.category.smelting.experience=%s XP
 gui.jei.category.fuel=Kraftstoffrezepte
+gui.jei.category.fuel.smeltCount.single=Schmelzt 1 Item
+gui.jei.category.fuel.smeltCount=Schmelzt %s Items
```

Comment:
But... the Duden does specifically claim it, as explained [here](#discussion_r82498771). This means the entire page applies to both forms of the verb, including the mentioned grammar.